docs/PROJECTS.md

```diff
@@ -0,0 +1,9 @@
+This page links other open source projects which can be usefull for your player implementation
+
+# UI over react-native-video
+ - [react-native-video-controls](https://github.com/itsnubix/react-native-video-controls): First reference player UI
+ - [react-native-media-console](https://github.com/criszz77/react-native-media-console): React-native-video-controls updated and rewritten in typescript
+ - [react-native-corner-video](https://github.com/Lg0gs/react-native-corner-video): A floating video player
+
+# Other tools
+ - [react-native-corner-video](https://github.com/tanguyantoine/react-native-music-control): A toolbox to control player over media session
```

Comment:
The name should be _react-native-music-control_

Reply:
OK, done, thank you